### ISSUE TITLE
50/50 component alt text field limit to 125 characters

### ISSUE THREAD

**Issue by koalie** [w3c/w3c-website#302]
**Describe the issue**
The parameters of the image in the 50/50 component allow for 300 characters for a "title", but only 125 characters for the "alt text". Why restrict so much? 

**URL**
For example, in https://beta.w3.org/about/ the alt text does not allow sufficient space to fit the alt text of the diagram used in the 50/50 section.

**Recommended solution**
If there isn't a rationale for limiting the amount of alt text to 125 characters, please augment it to a more reasonable size. 
For example, earlier today I had to describe a diagram elsewhere (not in a 50/50 component) and the character count was 531 with spaces.

**Additional context**
All information in an image needs to be provided to users who cannot see the image. Usually all text and diagram information in an image needs to be provided. It can be in alt text. Or it can be in the main text of the page. Or it can be in a separate page. In this case, providing a longdesc as a separate page isn't optimal.

**Comment by japuiglaucia** [comment]
50/50 component alt text field limit to 125 characters #302

**Comment by JamieC24** [comment]
This issue has been resolved locally, waiting on other work to be approved before deploying to dev for review
